src/main/java/edu/ie3/datamodel/models/result/thermal/DomesticHotWaterStorageResult.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models.result.thermal;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+import javax.measure.quantity.Dimensionless;
+import javax.measure.quantity.Energy;
+import javax.measure.quantity.Power;
+import tech.units.indriya.ComparableQuantity;
+
+/** Represents the results of Domestic Hot Water Storage */
+public class DomesticHotWaterStorageResult extends AbstractThermalStorageResult {
```

Comment:
The input model extends the `CylindricalStorageInput`. Why does the result not extend the `CylindricalStorageResult`?

Reply:
Liked the split into an abstract class more. Will do so for factory, input and results, so its aligned.